### ISSUE TITLE
frequent remote Docker eofs, timeouts, etc

### ISSUE THREAD

**Issue by mrkurt** [superfly/flyctl#404]
Here's one:
```
Error error building: failed to dial gRPC: cannot connect to the Docker daemon. Is 'docker daemon' running on this host?: read tcp 172.17.0.2:38412->213.188.198.250:10000: read: connection reset by peer
```
https://community.fly.io/t/remote-builder-fails-to-find-docker-daemon/1263

**Comment by jsierles** [comment]
FYI, this now happens every time I 'cold' deploy without the remote builder VM being booted up. Once it's up, deploys run fine.

**Comment by michaeldwan** [comment]
Fixed in #411
